pkg/apis/extensions/types.go

```diff
@@ -42,6 +42,15 @@ const (
 	SysctlsPodSecurityPolicyAnnotationKey string = "security.alpha.kubernetes.io/sysctls"
 )
 
+// The following identify resource constants for Kubernetes object types
+var (
+	// ReplicaSets, number
+	ResourceReplicaSets api.ResourceName = GroupName + "/replicasets"
+
+	// Deployments, number
+	ResourceDeployments api.ResourceName = GroupName + "/deployments"
```

Comment:
> @deads2k -- whats the graduation plan when things go from extensions to a new group? do i update old quotas, or do we write some converter?

I suspect you'll refactor quota in a way that allows two of these keys to point to the same backing evaluator.  Also, try to avoid "graduation", by choosing a correct API group to begin with.